jetty-osgi/test-jetty-osgi/src/test/java/org/eclipse/jetty/osgi/test/TestJettyOSGiAnnotationParser.java

```diff
@@ -64,8 +64,8 @@
         options.add(TestOSGiUtil.optionalRemoteDebug());
         options.add(CoreOptions.junitBundles());
         options.addAll(TestOSGiUtil.coreJettyDependencies());
-        options.add(mavenBundle().groupId("biz.aQute.bnd").artifactId("bndlib").versionAsInProject().start());
-        options.add(mavenBundle().groupId("org.ops4j.pax.tinybundles").artifactId("tinybundles").version("2.1.1").start());
+        options.add(mavenBundle().groupId("biz.aQute.bnd").artifactId("biz.aQute.bndlib").version("3.5.0").start());
```

Comment:
Shouldn't this be `.versionAsInProject()` instead of `.version("3.5.0")` ?

Reply:
Nope, see the comment in TestJettyOSGiBootWithJavaxWebSocket.java : for the tests that use tinybundles, we have to back down the version of bndlib for compatibility reasons.